README.md

```diff
@@ -29,7 +29,7 @@ Welcome all of you future star candidates who want to strive to attend the acade
   > As an academician <br>
 - [Aliza Nurfitrian Meizahra ](https://github.com/Alizaaaja4) [ALL]
   > As an academician <br> 
-  > Website link -> https://alizaaaja4.github.io/aliza/
+  > Website links -> https://alizaaaja4.github.io/aliza/
```

Comment:
rename to `Website Link`

Reply:
![image](https://github.com/Daskom-Lab/2023-Academy/assets/66113867/380c9c54-c478-4074-862a-f7c0c84b27bd)
and better if u change the default color from `black` to light color coz the background is dark